docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -565,6 +565,21 @@ data class Person @BsonCreator constructor (
 ): PanacheMongoEntity()
 ----
 
+The last option is to the use the link:https://kotlinlang.org/docs/reference/compiler-plugins.html#no-arg-compiler-plugin[no-arg] compiler plugin.
+This plugin needs to be configured with a lit of annotations, then it will generate a parameterless constructor for each class annotated with them.
```

Comment:
I would suggest:

```suggestion
This plugin is configured with a list of annotations, and the end result is the generation of no-args constructor for each class annotated with them.
```

Reply:
I prefere this one, sorry @gsmet :)
I found the sentence not very readable but I didn't find a better at this time, you know, my english ...

Reply:
I have the unfair advantage of being a native speaker :P